cli/ceph/osd/osd.py

```diff
@@ -67,3 +67,21 @@ def tree(self, epoch=None, states=None, **kw):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def out(self, id):
+        """Set osd out
+
+        Args:
+            id (str): osd id
+        """
+        cmd = f"{self.base_cmd} out {id}"
+        return self.execute(sudo=True, long_running=True, cmd=cmd)
+
+    def _in(self, id):
```

Comment:
Can we remove the _ here? Or any specific reason why its added Aditya?

Reply:
It wasn't allowing to use python keywords for method; hence I used _.

Reply:
Got it. @vamahaja need a help here, any other workarounds?

Reply:
@pranavprakash20, @vamahaja only suggested me to have it this method with _.

Reply:
@pranavprakash20 as `in` is inbuild python keyword, use `_in`